content/operate/rc/changelog/august-2023.md

```diff
@@ -33,6 +33,10 @@ If you'd like to use triggers and functions with a [Flexible subscription]({{< r
 
 For more information about triggers and functions, see the [triggers and functions documentation]({{< relref "/develop/interact/programmability/triggers-and-functions/" >}}).
```

Comment:
These links will need to be changed as well. I'm moving some, not all, T&F  content to

https://redis.io/docs/latest/operate/oss_and_stack/stack-with-enterprise/deprecated-features/

So, maybe point there?

Reply:
```suggestion
For more information about triggers and functions, see the [triggers and functions documentation]({{< relref "/operate/oss_and_stack/stack-with-enterprise/deprecated-features/triggers-and-functions/" >}}).
```